### PR TITLE
Allow space for >= 10 items in required scopes list

### PR DESCRIPTION
**Pull request by robertknight**
Tested by manually editing the padding to an equivalent value (2rem) on this test assignment: https://hypothesis.instructure.com/courses/121/assignments/852.

Fixes https://github.com/hypothesis/lms/issues/5785

### CODE REVIEW

File: lms/static/scripts/frontend_apps/components/OAuth2RedirectErrorApp.tsx [hunk 1]
```diff
@@ -64,7 +64,7 @@ export default function OAuth2RedirectErrorApp({
             A Canvas admin needs to edit {"Hypothesis's"} developer key and add
             these scopes:
           </p>
-          <ol className="px-4 list-decimal">
+          <ol className="px-8 list-decimal">
```
Comment (acelaya) [inline lms/static/scripts/frontend_apps/components/OAuth2RedirectErrorApp.tsx:67]: Perhaps I would add padding to the left only, as the container has its own "global" padding.

```suggestion
          <ol className="pl-8 list-decimal">
```

**Review by acelaya — APPROVED** [review]
The problem gets fixed with this, but I suggested adding the padding to the left only.

File: lms/static/scripts/frontend_apps/components/OAuth2RedirectErrorApp.tsx (hunk 1 above)
Comment (robertknight) [reply lms/static/scripts/frontend_apps/components/OAuth2RedirectErrorApp.tsx]: Yes. In this context left-padding makes sense. Note that for lists with translatable content, `padding-inline-start` (`ps-*` in Tailwind) is preferred as it works for both LTR and RTL languages.

Comment (acelaya) [reply lms/static/scripts/frontend_apps/components/OAuth2RedirectErrorApp.tsx]: Yes! Padding start is better. I overlooked tailwind supported this.

**Pull request merged**